src/ai-bundle/config/options.php

```diff
@@ -348,6 +348,17 @@
                             ->end()
                         ->end()
                     ->end()
+                    ->arrayNode('weaviate')
+                    ->normalizeKeys(false)
```

Comment:
Curious, why do we need this? Inline all others are using it too, but is it needed?

Reply:
Created issue https://github.com/symfony/ai/issues/404